src/components/Blog/BlogWidget.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box } from '@mui/material';
+import { WallettButtons } from '../Navbar/WalletButton';
+import { Widget } from '../Widgets';
+
+interface BlogWidgetProps {
+  fromChain?: number;
+  fromToken?: string;
+  toChain?: number;
+  toToken?: string;
+  fromAmount?: string;
+  allowChains?: string;
+}
+
+export const BlogWidget = ({
+  fromChain,
+  fromToken,
+  toChain,
+  toToken,
+  fromAmount,
+  allowChains,
+}: BlogWidgetProps) => {
+  const allowChainsArray = (allowChains || '')
+    .split(',')
+    .map((el) => parseInt(el))
+    .filter((num) => !isNaN(num));
+
+  return (
+    <>
+      <Box
+        sx={{ display: 'flex', justifyContent: 'flex-end', marginBottom: 3 }}
```

Comment:
Better to create a styled components into the related .style.ts file to follow the app structure

Reply:
Much better, thanks!